### PR TITLE
Update project-management-checklists.md

### PR DESCRIPTION
**Pull request by hhartingrex**
Added a few links. Thank you!

----
:books: Documentation preview :books:: https://civicactions-handbook--1052.org.readthedocs.build/en/1052/

### CODE REVIEW

File: practice-areas/project-management/project-management-checklists.md [hunk 1]
```diff
@@ -6,18 +6,19 @@ The Project Management practice area suggests leveraging the following checklist
 
 Wrap up the past month:
 
-1. Complete monthly contractual deliverables (monthly reporting, etc.)
-2. Run the "plan vs actual" report for the last month and in order to verify reporting accuracy
+1. Complete monthly contractual deliverables (monthly reporting, etc.), making sure they are in the shared folder
+2. Run the "plan vs actual" report for the last month and in order to verify reporting accuracy [See Unanet instructions](https://guidebook.civicactions.com/en/latest/practice-areas/project-management/project-management-checklists/)
```
Comment (dmundra) [inline practice-areas/project-management/project-management-checklists.md]: @hhartingrex it looks like you are linking the same page for the unanet instructions? Should it go into a different page and also can the link be relative like `pm-unanet-tasks.md`?

**Review by dmundra — CHANGES_REQUESTED** [review]
Looks like the links need some adjusting.

File: practice-areas/project-management/project-management-checklists.md (hunk 1 above)
Comment (hhartingrex) [reply practice-areas/project-management/project-management-checklists.md]: @dmundra Thank you! Didn't know about the relative trick. Fixing now!

**Review by farooqzakhilwal — APPROVED** [review]
looks great. I can't tell if the links were adjusted as per the above request.

**Review by dmundra — APPROVED** [review]
Links look good.

**Pull request merged**